campos_potenciais2.py

```
@@ -0,0 +1,278 @@
+    rob_ori = np.array(sim.getObjectOrientation(robotHandle, sim.handle_world))
+    robot_theta = rob_ori[2]
+    rel_goal_world = np.array([goal_world[0], goal_world[1]]) - rob_pos_world[:2]
+    # world -> robot rotation
+    c, s = cos(robot_theta), sin(robot_theta)
+    R_inv = np.array([[c, s], [-s, c]])
+    rel_goal_robot = R_inv @ rel_goal_world
+    F_att_robot = k_att * rel_goal_robot
+    return F_att_robot, rob_pos_world, robot_theta
+
+#--------------------------------------------Resultante das forcas ------------------------------------------------------------------------------
+def control_from_force_robot_frame(F_total, robot_theta, k_v=kv, k_w=kw, vlim=v_max, wlim=w_max):
```

Ruff (0.13.3)

Unused function argument: `robot_theta`

(ARG001)